koku/hcs/test/database/test_report_db_accessor.py

```diff
@@ -84,3 +88,62 @@ def test_data_hcs_customer(self, mock_dba_query, mock_fh_writer, mock_fh):
             )
             self.assertIn("acquiring marketplace data", _logs.output[0])
             self.assertIn("data found", _logs.output[1])
+
+    def test_handle_trino_external_error_with_no_such_key(self):
+        """Test handle_trino_external_error with NoSuchKey error."""
+        accessor = ReportDBAccessorBase(schema="test_schema")
+
+        with patch.object(accessor, "_execute_trino_raw_sql_query_with_description") as mock_retry, patch(
+            "masu.database.report_db_accessor_base.LOG"
+        ) as mock_log:
```

Comment:
As of Python 3.10, `with` statements can be broken across multiple lines using `()` for better readability..
```suggestion
        with (
            patch.object(accessor, "_execute_trino_raw_sql_query_with_description") as mock_retry,
            patch("masu.database.report_db_accessor_base.LOG") as mock_log
        ):
```